pkg/component/etcd/lease/values.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package lease
+
+import (
+	"k8s.io/apimachinery/pkg/types"
+)
+
+type Values struct {
```

Comment:
Instead of struct, should we make Values as `map[string]interface{}`? If we make it as struct, we need to pass separate structs to the `getMapFromEtcd` function for separate components in future. WDYT?

Reply:
Hm yes, got your point. However, `getMapFromEtcd` will only be there as long as we have not entirely transitioned all manifests to components. For the transitioning phase it should be fine to do so, wdyt?

The awkward point about `map[string]interface{}` is the missing type safety for values and that you need to create constants for the keys, so that other components can use them.

Reply:
Hmm.. Sadly, until getMapFromEtcd is removed completely, parameter list of `getMapFromEtcd` for each component type

Reply:
I added another `componentetcd.Values` struct which aggregates the different sub values. I hope that's fine.